cmd/config.go

```diff
@@ -138,115 +129,8 @@ func newConfigCommand(config *settings.Config) *cobra.Command {
-	var err error
-	var response *config.ConfigResponse
-	path := local.DefaultConfigPath
-	// First, set the path to configPath set by --path flag for compatibility
-	if configPath != "" {
-		path = configPath
-	}
-
-	// Then, if an arg is passed in, choose that instead
-	if len(opts.args) == 1 {
-		path = opts.args[0]
-	}
-
-	//if no orgId provided use org slug
-	values := pipeline.LocalPipelineValues()
-	if verboseOutput {
-		printValues(values)
-	}
-
-	var orgID string
-	orgID, _ = flags.GetString("org-id")
-	if strings.TrimSpace(orgID) != "" {
-		response, err = config.ConfigQuery(opts.rest, path, orgID, nil, pipeline.LocalPipelineValues())
-		if err != nil {
-			return err
-		}
-	} else {
-		orgSlug, _ := flags.GetString("org-slug")
-		orgs, err := GetOrgCollaborations(opts.cfg)
-		if err != nil {
-			fmt.Println(err.Error())
-		}
-		orgID = GetOrgIdFromSlug(orgSlug, orgs)
-		response, err = config.ConfigQuery(opts.rest, path, orgID, nil, pipeline.LocalPipelineValues())
-		if err != nil {
-			return err
-		}
-	}
-
-	// check if a deprecated Linux VM image is being used
-	// link here to blog post when available
-	// returns an error if a deprecated image is used
-	if !ignoreDeprecatedImages {
-		err := config.DeprecatedImageCheck(response)
-		if err != nil {
-			return err
-		}
-	}
-
-	if path == "-" {
-		fmt.Printf("\nConfig input is valid.\n")
-	} else {
-		fmt.Printf("\nConfig file at %s is valid.\n", path)
-	}
-
-	return nil
-}
-
-func processConfig(opts configOptions, flags *pflag.FlagSet) error {
-	paramsYaml, _ := flags.GetString("pipeline-parameters")
-	var response *config.ConfigResponse
-	var params pipeline.Parameters
-	var err error
-
-	if len(paramsYaml) > 0 {
-		// The 'src' value can be a filepath, or a yaml string. If the file cannot be read successfully,
-		// proceed with the assumption that the value is already valid yaml.
-		raw, err := os.ReadFile(paramsYaml)
-		if err != nil {
-			raw = []byte(paramsYaml)
-		}
-
-		err = yaml.Unmarshal(raw, &params)
-		if err != nil {
-			return fmt.Errorf("invalid 'pipeline-parameters' provided: %s", err.Error())
-		}
-	}
-
-	//if no orgId provided use org slug
-	values := pipeline.LocalPipelineValues()
-	printValues(values)
-
-	orgID, _ := flags.GetString("org-id")
-	if strings.TrimSpace(orgID) != "" {
-		response, err = config.ConfigQuery(opts.rest, opts.args[0], orgID, params, values)
-		if err != nil {
-			return err
-		}
-	} else {
-		orgSlug, _ := flags.GetString("org-slug")
-		orgs, err := GetOrgCollaborations(opts.cfg)
-		if err != nil {
-			fmt.Println(err.Error())
-		}
-		orgID = GetOrgIdFromSlug(orgSlug, orgs)
-		response, err = config.ConfigQuery(opts.rest, opts.args[0], orgID, params, values)
-		if err != nil {
-			return err
-		}
-	}
-
-	fmt.Print(response.OutputYaml)
-	return nil
-}
-
-func packConfig(opts configOptions) error {
-	tree, err := filetree.NewTree(opts.args[0])
+func packConfig(args []string) error {
+	tree, err := filetree.NewTree(args[0])
```

Comment:
Should we verify that len(args) > 0 ? 
Unless this is being done somewhere I did not find

Reply:
I believe Line 37 `Args:        cobra.ExactArgs(1),` does this for us - otherwise it outputs the help output.